UPGRADE-4.4.md

```diff
@@ -168,6 +168,8 @@ Mailer
 ------
 
  * [BC BREAK] Changed the DSN to use for disabling delivery (using the `NullTransport`) from `smtp://null` to `null://null` (host doesn't matter).
+ * [BC BREAK] Renamed class `SmtpEnvelope` to `Envelope`.
```

Comment:
Should we also talk about `DelayedSmtpEnvelope` (it was renamed to `DelayedEnvelope`)?

Reply:
You're right, I don't use this class so I didn't notice its name has changed